src/WebUI/KubeFactory.ts

```diff
@@ -9,9 +9,28 @@ export class KubeFactory {
         // Default implementation: Do nothing
     }
 
+    public static telemetryService: ITelemetryService;

```

Comment:
should we follow pattern, where we say, set/get default(), so the variable will be private rather public. same way we did for RD service?

Reply:
Addressing all this in a follow up PR. I've to add all click telemetries as well